window/src/os/x11/window.rs

```diff
@@ -1543,8 +1543,8 @@ impl XWindowInner {
             min_aspect_den: 0,
             max_aspect_num: 0,
             max_aspect_den: 0,
-            base_width: 0,
-            base_height: 0,
+            base_width: base_width.into(),
```

Comment:
I don't see a macOS equivalent of this, but perhaps it can be emulated by calculating https://developer.apple.com/documentation/appkit/nswindow/1419124-contentlayoutrect?language=objc appropriately